s29093-AlicjaChelminiak/zajecia3/zadanie1.4.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ILOSC_PLIKOW=$(ls -1 | wc -l)
```

Comment:
Do poprawy. To zwraca ilość obiektów w katalogu. Jesli w katalogu będzie coś poza plikami np. zagnieżdżone katalogi to algorytm je zliczy jako pliki.